tests/core/full_node/test_conditions.py

```diff
@@ -148,20 +157,6 @@ async def test_valid_my_id(self, bt):
         conditions = Program.to(assemble(f"(({ConditionOpcode.ASSERT_MY_COIN_ID[0]} 0x{coin.name().hex()}))"))
         await check_conditions(bt, conditions)
 
-    @pytest.mark.asyncio
-    async def test_invalid_seconds_absolute(self, bt):
-        # TODO: make the test suite not use `time.time` so we can more accurately
-        # set `time_now` to make it minimal while still failing
```

Comment:
I fixed this TODO by forcing the time stamps on the generated blocks (see `genesis_timestamp` in the call to `get_consecutive_blocks()` above). Using the wall clock time won't work in the parameters because the name of the test would then depend on the wall clock time, which doesn't work when running tests in parallel